[2025-02-23T15:52:38.628+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:38.667+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-06T00:00:00+00:00 [queued]>
[2025-02-23T15:52:38.694+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-06T00:00:00+00:00 [queued]>
[2025-02-23T15:52:38.696+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:38.804+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-06 00:00:00+00:00
[2025-02-23T15:52:38.814+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1445) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:38.822+0000] {standard_task_runner.py:72} INFO - Started process 1471 to run task
[2025-02-23T15:52:38.822+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-06T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmp8i49aqdy']
[2025-02-23T15:52:38.826+0000] {standard_task_runner.py:105} INFO - Job 34: Subtask third_task
[2025-02-23T15:52:38.916+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-06T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:39.026+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:00:00+00:00'
[2025-02-23T15:52:39.027+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:39.050+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:39.052+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-23T15:52:39.068+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:39.070+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-23T15:52:39.071+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:39.101+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:39.102+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-06T00:00:00+00:00, execution_date=20250206T000000, start_date=20250223T155238, end_date=20250223T155239
[2025-02-23T15:52:39.244+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:39.255+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:14.119+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:14.164+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-06T00:00:00+00:00 [queued]>
[2025-02-24T04:47:14.180+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-06T00:00:00+00:00 [queued]>
[2025-02-24T04:47:14.180+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:14.316+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-06 00:00:00+00:00
[2025-02-24T04:47:14.332+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=866) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:14.334+0000] {standard_task_runner.py:72} INFO - Started process 886 to run task
[2025-02-24T04:47:14.333+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-06T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpgtsv6jam']
[2025-02-24T04:47:14.341+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask third_task
[2025-02-24T04:47:14.456+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-06T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:14.575+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:00:00+00:00'
[2025-02-24T04:47:14.576+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:14.595+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:14.597+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-24T04:47:14.609+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:14.611+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-24T04:47:14.612+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:14.642+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:14.643+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-06T00:00:00+00:00, execution_date=20250206T000000, start_date=20250224T044714, end_date=20250224T044714
[2025-02-24T04:47:14.719+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:14.740+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
